### DISCUSSION TITLE
[QUESTION] when i delete pod data,i found the size in server are not reduce,how can i reduce replicas catalog size in server?

### DISCUSSION THREAD

**Discussion by kikyoulg** [longhorn/longhorn#5302] [Q&A]
## Question
> Suggest to use https://github.com/longhorn/longhorn/discussions to ask questions.

## Environment

 - Longhorn version:
 - Kubernetes version:
 - Node config
   - OS type and version
   - CPU per node:
   - Memory per node:
   - Disk type
   - Network bandwidth and latency between the nodes:
 - Underlying Infrastructure (e.g. on AWS/GCE, EKS/GKE, VMWare/KVM, Baremetal):

**Comment by weizhe0422** [comment]
If you are using Longhorn v1.4.0, there is a new feature: [Trim Filesystem](https://longhorn.io/docs/1.4.0/volumes-and-nodes/trim-filesystem/#trim-the-filesystem-in-a-longhorn-volume) to reclaim space which occupied by removed files of the filesystem.

**Reply by kikyoulg** [reply]
it's a new version and only can be installed in kubernetes 1.21,but my kubernetes version is 1.20.2.....
